### ISSUE TITLE
External links to a more detailed example of each fallacy

### ISSUE THREAD

**Issue by LukeStonehm** [LukeStonehm/LogicalDefence#6]
It has been suggested (through a review on the Play Store) that I add an external link to a more detailed for each fallacy. I will start to compile a list of examples, although this is a momentous task. Any help would be appreciated. Going to create a Google Sheet with each fallacy and the link so that I can crowd-source examples :)

**Comment by xwbfozuh** [comment]
Wikipedia has a [huge list](https://en.wikipedia.org/wiki/List_of_fallacies). Linking to them could open it in a browser or the Wikipedia app if the user has it installed.

**Comment by LukeStonehm** [comment]
@xwbfozuh Thank you for this. I think this may be the best approach. I personally have the Wikipedia app installed (it is fantastic, so much better than using the web browser)

This will be next weekends goal I think :)

**Comment by wolfgangus** [comment]
I think I can't really see all of the comments on this issue, but if you are having some trouble, you could take a look at rationalwiki.org. They do tend to have a sort of anti-religion bias, but technically there's nothing wrong with their wiki, apart from the fact that they could cover a lot more non-religion related stuff.

On the bottom right of their main page you could see that they have a logic portal. These links are a collection of some fallacies which might be relevant.

Wikipedia also has a article called list_of_fallacies. I think you already know about that though.

**Comment by wolfgangus** [comment]
Anyway, if you are going to do more apps, I have a few suggestions, which are missing in the android free software community...

**Comment by LukeStonehm** [comment]
I am aware of the the list of fallacies on Wiki, I will be updating the app to include more of the formal fallacies soon.

What free software android apps are missing? I am always willing to make more open source projects :) I am attempting to build a strong portfolio.

**Comment by wolfgangus** [comment]
Hey, I replied to your message via e-mail..I'm not sure if you got it or not...because the comment isn't posted here :)

**Comment by sgeto** [comment]
It would also be nice if you could provide a printable version of all fallacies so we could use it as teaching material.

**Comment by LukeStonehm** [comment]
@sgeto I have been thinking about making a web version of the Logical Defence, and hosting on my website.. When I have time in Dec, I will look into doing it :smile:

**Comment by sgeto** [comment]
Sounds awesome! Thanks. Looking forward.

**Comment by timofonic** [comment]
Are the plans still in consideraron?

**Comment by LukeStonehm** [comment]
Yes indeed they are, I need to find energy to get started. Been planning a fresh look for the app, so I may incorporate this feature into it. Going to setup a Milestone 😄
